Dockerfile

```diff
@@ -1,19 +1,21 @@
-FROM ubuntu:disco
+FROM ubuntu:focal
 
 ARG MAKEFLAGS=-j2
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
When installing packages, package tzdata requires user input to select a timezone. Adding this argument to prevent the docker build to hang waiting for input.